libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -154,6 +161,8 @@ class AP_MotorsHeli_RSC {
     float           _governor_output;             // governor output for rotor speed control
     bool            _governor_engage;             // RSC governor status flag for soft-start
     bool            _use_bailout_ramp;            // true if allowing RSC to quickly ramp up engine
+    bool            _in_autorotaion;              // true if vehicle is currently in an autorotaion
+    int16_t         _rsc_arot_bailout_sp;         // the microscond signal sent to external governor to signal that autorotation bailout ramp should be used
```

Comment:
I'm probably repeating myself by now but is it a PWM or a percentage?  If it's a percentage we sometimes put "_pct" at the end of the variable name.